src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -3921,7 +3921,7 @@ push that
                     if (Constants.kGlobalScope == classId)
                     {
                         procName = exe.procedureTable[blockId].procList[procId].name;
-                        CodeBlock codeblock = core.GetCodeBlock(core.CodeBlockList, blockId);
+                        CodeBlock codeblock = ProtoCore.Utils.CoreUtils.GetCodeBlock(exe.CodeBlocks, blockId);
```

Comment:
Yea that should be done at somepoint. Perhaps not in this submission, It can potentially destabilize.